src/Monolog/Handler/ErrorLogHandler.php

```diff
@@ -71,13 +71,14 @@ protected function getDefaultFormatter(): FormatterInterface
      */
     protected function write(array $record)
     {
-        if ($this->expandNewlines) {
-            $lines = preg_split('{[\r\n]+}', (string) $record['formatted']);
-            foreach ($lines as $line) {
-                error_log($line, $this->messageType);
-            }
-        } else {
+        if (!$this->expandNewlines) {
```

Comment:
It's all about readability.  Logically they are the same.

In the same way you see no value in the negititive, I see no way in this section of code how truths is better.  In this case we have blocks within blocks within blocks.   Not only does it make it hard to read it's also harder to follow.   If you exit out early you don't need to worry about such things. 

If the bulk of code fell inside the negitive block of code, I'd have submitted a PR in the reverse. 

But this isn't my project, just thought I'd pass this along while I was in there.